src/controller/content/content.service.ts

```diff
@@ -623,13 +623,7 @@ export class ContentService {
         delete CodingQuestions.CodingQuestion
         return { ...CodingQuestions, ...codingDetails }
       })
-
-      chapterDetails.Forms = chapterDetails?.Forms.map((Forms) => {
```

Comment:
could you please confirm if the form chapters are currently operational?

Reply:
Yes